h/static/scripts/annotator/anchoring/pdf.coffee

```diff
@@ -249,6 +251,10 @@ exports.describe = (root, range, options = {}) ->
   startTextLayer = getNodeTextLayer(range.start)
   endTextLayer = getNodeTextLayer(range.end)
 
+  if startTextLayer is null
```

Comment:
The range object that Annotator's `Range.BrowserRange.normalize()` returns sometimes has `range.start == null` (whether intentional or a bug, I don't know). This results in our `pdf.coffee` calling `getNodeTextLayer(null)` which crashes.